### ISSUE TITLE
Simulation Experiment view: to reload a SED-ML results in curves not being plotted with all of their original settings

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#2176]
Open the `models/tests/sedml/noble_1962_local.sedml` file. If you run it, you should get something like:

![Screen Shot 2019-10-03 at 16 20 00](https://user-images.githubusercontent.com/602265/66097152-d08c8c00-e5f9-11e9-9125-9e1ffb3ecae3.png)

Then, if you reload the SED-ML and rerun it, you should now get something like:

![Screen Shot 2019-10-03 at 16 20 09](https://user-images.githubusercontent.com/602265/66097157-d5514000-e5f9-11e9-853c-218d1b108ed0.png)

In other words, none of the curves are rendered using all of their original settings anymore.

FWIW, this problem appeared sometime between snapshots `2019-02-01` and `2019-03-07`.
